flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/NettyServer.java

```diff
@@ -170,8 +171,8 @@ public void initChannel(SocketChannel channel) throws Exception {
 
 		localAddress = (InetSocketAddress) bindFuture.channel().localAddress();
 
-		long end = System.currentTimeMillis();
-		LOG.info("Successful initialization (took {} ms). Listening on SocketAddress {}.", (end - start), bindFuture.channel().localAddress().toString());
+		final long duration = (start - System.nanoTime()) / 1_000_000;
+		LOG.info("Successful initialization (took {} ms). Listening on SocketAddress {}.", duration, localAddress);
```

Comment:
Is this should be `final long duration = (System.nanoTime() - start) / 1_000_000;`

Reply:
You are absolutely right, thanks for catching this!